app/trip/[id]/components/trip-details-header.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  Heading,
+  Tag,
+  Box,
+  Flex,
+  useColorModeValue,
+  Image as ChakraImage,
+  Link,
+  IconButton
+} from '@chakra-ui/react'
+import Image from 'next/image'
+import { useRouter } from 'next/navigation'
+import { FiEdit3, FiShare2, FiTrash2 } from 'react-icons/fi'
+import { MdManageAccounts, MdAccountCircle } from 'react-icons/md'
+import { formatDateToDayMonthYear } from '@/libs/utils'
+
+type TripDetailsHeaderProps = {
+  id: string
+  image: string | null | undefined
+  title: string
+  dateFrom: string | null | undefined
+  dateTo: string | null | undefined
+  users: {
+    id: number | undefined
+    image: string | null | undefined
+  }[]
+
+  tags: { id: number | undefined; name: string | undefined }[]
+}
+
+export const TripDetailsHeader = ({
+  id,
+  image,
+  title,
+  dateFrom,
+  dateTo,
+  tags,
+  users
+}: TripDetailsHeaderProps) => {
+  const router = useRouter()
+
+  const imageSrc = useColorModeValue(
+    '/images/no_image_light.jpg',
+    '/images/no_image_dark.jpg'
+  )
+
+  const color = useColorModeValue('black', 'gray.300')
+  const tagBgColor = useColorModeValue('primary.700', 'primary.800')
+
+  return (
+    <Box>
+      <Box
+        w={{ base: 'auto', md: '100%' }}
+        h={{ base: '164px', md: '288px' }}
+        position="relative"
+        ml={{ base: 'calc(50% - 50vw)', md: '0px' }}
+        mr={{ base: 'calc(50% - 50vw)', md: '0px' }}
+      >
+        <Image
+          src={image || imageSrc}
```

Comment:
It looks great with an image, but if users haven't inserted an image, they will see the default image, which looks a bit awkward. The icon in the center seems a little too big. It might be better to have the icon nicely fit in the middle with some paddings around it and you might want to lower the opacity so that it won't stand out too much??
<img width="1215" alt="Screenshot 2023-12-22 at 10 21 03" src="https://github.com/tabi-memo/frontend/assets/70562492/3e712859-e7a5-4bb4-859a-10a2ce22928d">

Reply:
Thanks! I made different image for this page
![screencapture-localhost-3000-trip-24cfbb19-9939-4965-aeca-68c5ee1759e1-2023-12-22-19_56_05 (1)](https://github.com/tabi-memo/frontend/assets/66394413/516b6fcf-2cf8-4d0b-bd59-eea73b9e4628)